src/templates/SimpleCoo/SimpleCooTemplate.test.tsx

```diff
@@ -36,6 +42,12 @@ describe("simple coo v3", () => {
     render(<SimpleCooTemplate document={SimpleCooSampleV3} handleObfuscation={() => {}} />);
     expect(screen.getByTestId("signature-second")).toBeInTheDocument();
   });
+
+  // eslint-disable-next-line jest/no-commented-out-tests
+  // it("should be able to wrap v3", async () => {
+  //   const wrappedDocument = await wrapDocumentV3(SimpleCooSampleV3);
+  //   expect(utils.isWrappedV3Document(wrappedDocument)).toBe(true);
+  // }); // TODO: simple coo has no v3 schema defined at schemata yet -> https://schemata.openattestation.com
```

Comment:
Is this part intentionally commented out or left here by accident? Based on the `TODO` it looks like it's intentional to me but just in case I'm wrong.

Reply:
yeah intentional